patches/minecraft/net/minecraft/world/level/entity/PersistentEntitySectionManager.java.patch

```diff
@@ -0,0 +1,21 @@
+--- a/net/minecraft/world/level/entity/PersistentEntitySectionManager.java
++++ b/net/minecraft/world/level/entity/PersistentEntitySectionManager.java
+@@ -72,7 +_,17 @@
+       return this.m_157538_(p_157534_, false);
+    }
+ 
+-   private boolean m_157538_(T p_157539_, boolean p_157540_) {
++   public boolean addNewEntityWithoutEvent(T p_157534_) {
++      return this.addEntityWithoutEvent(p_157534_, false);
++   }
++
++   private boolean m_157538_(T p_157539_, boolean p_157540_)
++   {
```

Comment:
Bracket should be on previous line still

Reply:
Oops.